server/monitor/src/main/java/org/apache/accumulo/monitor/EmbeddedWebServer.java

```diff
@@ -118,19 +98,17 @@ public void start() {
     }
   }
 
-  public void stop() {
+  private void stop() {
```

Comment:
I was trying to clean up the embedded web server, and simplify its abstraction around Jetty. We never call stop outside the method. I marked it private to verify and left it that way. This is an internal class, so there's no reason it should be public if it doesn't need to be. I would like to do further refactoring and simplification of the Java portions of the monitor service... but that can happen in a future task.